src/openvic-simulation/pop/Pop.cpp

```diff
@@ -114,6 +114,16 @@ void Pop::setup_pop_test_values(IssueManager const& issue_manager) {
 	luxury_needs_fulfilled = test_range();
 }
 
+void Pop::convert_to_equivalent() {
+	PopType const* const equivalent = get_type()->get_equivalent();
+	if(equivalent == nullptr) {
+		//error?
```

Comment:
Make this function return a bool: false if this error occurs, true otherwise, and `Logger::error` a message. Pass the return value of this function up through the functions that call it.